Flexx/Classes/Flexx.swift

```diff
@@ -253,17 +263,18 @@ public class Flexx {
     /// to read backup file. If backup file is empty we switch the locale
     /// to default locale and read the backup file for it.
     private func readLocaleFile(fileName: String, domain: String) -> Data {
+        let languageCode = Flexx.shared.checkLocaleId(localeId: fileName)
```

Comment:
Why don't we provide the language code as `fileName` param and avoid changes here?

Reply:
refactor